nomad/resource_job_test.go

```diff
@@ -885,6 +886,7 @@ resource "nomad_job" "test" {
 				healthy_deadline = "6m"
 				progress_deadline = "11m"
 				auto_revert = true
+        auto_promote = true
```

Comment:
this test was looking at new config for the 0.9.0 API... `auto_promote` should be part of a separate test for 0.9.3. such a test should verify that this option is successfully being submitted to Nomad.